src/lib/y2partitioner/actions/controllers/encryption.rb

```diff
@@ -55,6 +55,12 @@ class Encryption < Base
         # @return [Array<Y2Storage:.EncryptionProcesses::Apqn>]
         attr_accessor :apqns
 
+        # @return [String] Label for the encryption device if the method supports setting one
+        attr_accessor :label
+
+        # @return [String] PBKDF for the LUKS2 device
```

Comment:
NP: I prefer to see the full name instead of the acronym at least once. I mean:

```suggestion
        # @return [String] Password-Based Key Derivation Function (PBKDF) for the LUKS2 device
```